### PR TITLE
feat: Export custom forms field in csv, pdf for attendee

### PR DESCRIPTION
**Pull request by codedsun**
Fixes https://github.com/fossasia/open-event-frontend/issues/4615

#### Short description of what this resolves:
Exported custom forms field in csv, pdf for attendee

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: app/templates/pdf/attendees_pdf.html [hunk 1]
```diff
@@ -107,34 +107,33 @@ <h3 style="text-align:center;">{{ holders[0].event.name }} - {{ ("Attendees List
...
-                    <strong>
-                        {{ holder.firstname }}<br>
-                        {{ holder.lastname }}<br>
-                        {{ holder.email }}<br>
-                    </strong>
+                 </td>
+                 <td><br>
+                     {% for field in custom_forms %}
+                        {% if field.is_complex %}-->
+                            {{ holder.complex_field_values | attr(field.field_identifier) }} {{ holder.complex_field_values | attr(field.field_identifier) }} <br>
```
Comment (codedsun) [inline app/templates/pdf/attendees_pdf.html:120]: @iamareebjamal - Need your help here, how do I apply a condition when the attribute is not present.

Comment (iamareebjamal) [reply app/templates/pdf/attendees_pdf.html:120]: if field.is_complex and holder.complex_field_values

Comment (codedsun) [reply app/templates/pdf/attendees_pdf.html:120]: image

Like an arrows are developing here I want this to be `''`

Comment (iamareebjamal) [reply app/templates/pdf/attendees_pdf.html:120]: Remove `-->`

File: app/api/helpers/tasks.py [hunk 2]
```diff
@@ -501,6 +502,9 @@ def export_order_pdf_task(self, event_id):
 @celery.task(base=RequestContextTask, name='export.attendees.csv', bind=True)
 def export_attendees_csv_task(self, event_id):
     attendees = db.session.query(TicketHolder).filter_by(event_id=event_id)
+    custom_forms = db.session.query(CustomForms).filter_by(
```
Comment (iamareebjamal) [inline app/api/helpers/tasks.py:505]: Filter out deleted custom forms

File: app/templates/pdf/attendees_pdf.html (hunk 1 above)
Comment (codedsun) [reply app/templates/pdf/attendees_pdf.html:120]: O my eyes 👀

File: app/api/helpers/tasks.py (hunk 2 above)
Comment (codedsun) [reply app/api/helpers/tasks.py:505]: This model extends db.Model and not soft deleted model

File: app/templates/pdf/attendees_pdf.html (hunk 1 above)
Comment (codedsun) [reply app/templates/pdf/attendees_pdf.html:120]: @iamareebjamal - Still the values are not fetched by ` holder.complex_field_values | attr(field.field_identifier)` will check this

File: app/api/helpers/tasks.py (hunk 2 above)
Comment (iamareebjamal) [reply app/api/helpers/tasks.py:505]: OK

File: app/templates/pdf/attendees_pdf.html (hunk 1 above)
Comment (codedsun) [reply app/templates/pdf/attendees_pdf.html]: Fixed

**Comment by codedsun** [comment]
@iamareebjamal Tested! Shall I provide you the csv and pdf?

**Comment by iamareebjamal** [comment]
Yes, and update the screenshots

**Comment by iamareebjamal** [comment]
And fix the build

**Comment by codedsun** [comment]
CSV
image

PDF
image

**Comment by iamareebjamal** [comment]
Which one is custom among the fields?

**Comment by codedsun** [comment]
> Which one is custom among the fields?

Github Link, Mobile 

Can be seen on last 2nd order

**Comment by codedsun** [comment]
image

Last 2nd order in pdf.

**Comment by codedsun** [comment]
@iamareebjamal Review!

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**